build.log

```diff
@@ -0,0 +1,141 @@
+do so (now or later) by using -c with the switch command. Example:
+
+  git switch -c <new-branch-name>
+
+Or undo this operation with:
+
+  git switch -
+
+Turn off this advice by setting config variable advice.detachedHead to false
+
+HEAD is now at 91f1a37 Roll abseil_revision 300fafa291..e5e2a9de56
+['git', 'init'] (cwd=PosixPath('/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/sbuild/native/pdfium/third_party/fast_float/src'))
+Initialized empty Git repository in /Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/sbuild/native/pdfium/third_party/fast_float/src/.git/
+['git', 'remote', 'add', 'origin', 'https://chromium.googlesource.com/external/github.com/fastfloat/fast_float'] (cwd=PosixPath('/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/sbuild/native/pdfium/third_party/fast_float/src'))
+['git', 'fetch', '--depth', '1', 'origin', 'cb1d42aaa1e14b09e1452cfdef373d051b8c02a4'] (cwd=PosixPath('/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/sbuild/native/pdfium/third_party/fast_float/src'))
+From https://chromium.googlesource.com/external/github.com/fastfloat/fast_float
+ * branch            cb1d42aaa1e14b09e1452cfdef373d051b8c02a4 -> FETCH_HEAD
+['git', 'checkout', 'FETCH_HEAD'] (cwd=PosixPath('/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/sbuild/native/pdfium/third_party/fast_float/src'))
+Note: switching to 'FETCH_HEAD'.
+
+You are in 'detached HEAD' state. You can look around, make experimental
+changes and commit them, and you can discard any commits you make in this
+state without impacting any branches by switching back to a branch.
+
+If you want to create a new branch to retain commits you create, you may
+do so (now or later) by using -c with the switch command. Example:
+
+  git switch -c <new-branch-name>
+
+Or undo this operation with:
+
+  git switch -
+
+Turn off this advice by setting config variable advice.detachedHead to false
+
+HEAD is now at cb1d42a version 7
+Downloading generate_shim_headers.py at revision 138.0.7191.0
+Traceback (most recent call last):
+  File "/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/setupsrc/pypdfium2_setup/build_native.py", line 407, in <module>
+    main_cli()
+  File "/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/setupsrc/pypdfium2_setup/build_native.py", line 403, in main_cli
+    main(**vars(args))
+  File "/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/setupsrc/pypdfium2_setup/build_native.py", line 336, in main
+    setup_compiler(config, compiler, clang_path)
+  File "/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/setupsrc/pypdfium2_setup/build_native.py", line 294, in setup_compiler
+    clang_version = _get_clang_ver(clang_path)
+                    ^^^^^^^^^^^^^^^^^^^^^^^^^^
+  File "/Users/karlchow/Desktop/code/pypdfium2-test/pypdfium2/setupsrc/pypdfium2_setup/build_native.py", line 281, in _get_clang_ver
+    from packaging.version import Version
+ModuleNotFoundError: No module named 'packaging'
```

Comment:
As the error says, this code needs the `packaging` module. It is listed in our `pyproject.toml` build system deps and pretty basic anyway so not sure why it was not available in your setup.